engine/provider.ftl

```diff
@@ -110,6 +110,39 @@
     [/#list]
 [/#macro]
 
+[#macro includeTestCaseConfiguration provider testCase ]
+    [#if isConfigurationIncluded([provider, testCase]) ]
+        [#return]
+    [/#if]
+
+    [#local templates = []]
+    [#list ["testcase"] as level ]
```

Comment:
```"testcase"```
I can't find conclusive advice anywhere as to whether this will work without camel-case or not (as it was defined). Probably works fine, but bringing this to your attention.

Reply:
Same in the list below also.

Reply:
the convention for template and macro dynamic loading seems to be lowercase based on the component dynamic loading. I think there are limits to the characters you can use in function/macro names